.env.testnet-aws

```diff
@@ -0,0 +1,12 @@
+NETWORK_NAME=testnet-aws
```

Comment:
It would be nice adding this value (testnet-aws) into the workflows/consecutive.yml to be able to start it manually

Reply:
Done